webapp/channels/src/components/suggestion/suggestion_box/suggestion_box.jsx

```diff
@@ -374,7 +374,11 @@ export default class SuggestionBox extends React.PureComponent {
 
         let prefix;
         let keepPretext = false;
-        if (pretext.toLowerCase().endsWith(matchedPretext.toLowerCase())) {
+
+        const normalizedPretext = pretext.replace(/＠/g, '@').toLowerCase();
+        const normalizedMatchedPretext = matchedPretext.replace(/＠/g, '@').toLowerCase();
```

Comment:
Are you sure we need this? As long as `matchedPretext` contains the full-width at-sign and the text that's replacing it contains the regular at-sign, I don't think we should need to replace this here.

Generally, the `SuggestionBox` shouldn't need to know anything about individual providers like the `AtMentionProvider` or have special handling for any of them. There's some places where we might have done that previously, but I don't think we should need any to support full-width at-signs